src/Microsoft.ApiDesignGuidelines.Analyzers/Core/ParameterNamesShouldMatchBaseDeclaration.cs

```diff
@@ -29,10 +34,113 @@ public abstract class ParameterNamesShouldMatchBaseDeclarationAnalyzer : Diagnos
                                                                              helpLinkUri: null,     // TODO: add MSDN url
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
+        private static readonly Action<SymbolAnalysisContext> AnalyzeMethodSymbolAction = AnalyzeMethodSymbol;
```

Comment:
Is this field required? Can't you directly pass `AnalyzeMethodSymbol` to `RegisterSymbolAction`?